public/app/plugins/panel/canvas/ConnectionAnchors.tsx

```diff
@@ -0,0 +1,139 @@
+import { css } from '@emotion/css';
+import React, { useRef } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { useStyles2 } from '@grafana/ui';
+import { ConnectionCoordinates } from 'app/features/canvas';
+
+type Props = {
+  setRef: (anchorElement: HTMLDivElement) => void;
+  handleMouseLeave: (event: React.MouseEvent<Element, MouseEvent> | React.FocusEvent<HTMLDivElement, Element>) => void;
+};
+
+export const CONNECTION_ANCHOR_DIV_ID = 'connectionControl';
+
+export const ConnectionAnchors = ({ setRef, handleMouseLeave }: Props) => {
+  const highlightEllipseRef = useRef<HTMLDivElement>(null);
+  const styles = useStyles2(getStyles);
+  const halfSize = 2.5;
+  const halfSizeHighlightEllipse = 5.5;
+  const anchorImage =
+    'data:image/svg+xml;base64,PCFET0NUWVBFIHN2ZyBQVUJMSUMgIi0vL1czQy8vRFREIFNWRyAxLjEvL0VOIiAiaHR0cDovL3d3dy53My5vcmcvR3JhcGhpY3MvU1ZHLzEuMS9EVEQvc3ZnMTEuZHRkIj48c3ZnIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHdpZHRoPSI1cHgiIGhlaWdodD0iNXB4IiB2ZXJzaW9uPSIxLjEiPjxwYXRoIGQ9Im0gMCAwIEwgNSA1IE0gMCA1IEwgNSAwIiBzdHJva2Utd2lkdGg9IjIiIHN0eWxlPSJzdHJva2Utb3BhY2l0eTowLjQiIHN0cm9rZT0iI2ZmZmZmZiIvPjxwYXRoIGQ9Im0gMCAwIEwgNSA1IE0gMCA1IEwgNSAwIiBzdHJva2U9IiMyOWI2ZjIiLz48L3N2Zz4=';
+
+  const onMouseEnterAnchor = (event: React.MouseEvent) => {
+    if (!(event.target instanceof HTMLImageElement)) {
+      return;
+    }
+
+    if (highlightEllipseRef.current && event.target.style) {
+      highlightEllipseRef.current.style.display = 'block';
+      highlightEllipseRef.current.style.top = `calc(${event.target.style.top} - ${halfSizeHighlightEllipse}px)`;
+      highlightEllipseRef.current.style.left = `calc(${event.target.style.left} - ${halfSizeHighlightEllipse}px)`;
+    }
+  };
+
+  const onMouseEnterHighlightElement = () => {
+    // TODO: Implement drawing a connection here (or in scene selecto implementation)
+  };
+
+  const onMouseLeaveHighlightElement = () => {
+    if (highlightEllipseRef.current) {
+      highlightEllipseRef.current.style.display = 'none';
+    }
+  };
+
+  const connectionAnchorAlt = 'connection anchor';
+
+  // Unit is percentage from the middle of the element
+  // 0, 0 middle; -1, -1 bottom left; 1, 1 top right
+  const ANCHORS = [
```

Comment:
nice! -- could be good to have this come from the element definition, but this is a good start